lsp_bridge.py

```diff
@@ -191,6 +191,13 @@ def _open_file(self, filepath):
                 return False
             
             lang_server_info = load_single_server_info(single_lang_server)
+
+            windows_user_emacs_dir_keyword = "%USER_EMACS_DIRECTORY%"
+            if windows_user_emacs_dir_keyword in lang_server_info["command"][0]:
+                user_emacs_dir = get_emacs_func_result("get-user-emacs-directory").replace("/", "\\")
```

Comment:
Why need replace / with \\ ? 

\\ is window path splitter, It won't works with Linux.